packages/example-next-js/pages/buttons.tsx

```diff
@@ -0,0 +1,271 @@
+import "bootstrap/dist/css/bootstrap.css";
+import type { NextPage } from "next";
+import { GlowSignInButton } from "../components/GlowSignInButton";
+import styles from "../styles/Buttons.module.scss";
```

Comment:
Haha I find CSS modules pretty frustrating in terms of DX. But it's fine since this isn't what we are going to publish for reuse.